src/libraries/StateLibrary.sol

```diff
@@ -219,7 +219,7 @@ library StateLibrary {
      * @notice Retrieves the position information of a pool without needing to calculate the `positionId`.
      * @dev Corresponds to pools[poolId].positions[positionId]
      * @param poolId The ID of the pool.
-     * @param owner The owner of the liquidity position.
+     * @param posm The position manager.
```

Comment:
```suggestion
     * @param owner The owner contract of the liquidity position. Typically a periphery contract like PositionManager
```

Reply:
reverting back to the `owner` naming, and just improving the natspec is good enough!